modules/roles/tx.go

```diff
@@ -0,0 +1,80 @@
+package roles
+
+import (
+	"github.com/tendermint/go-wire/data"
+
+	"github.com/tendermint/basecoin"
+	"github.com/tendermint/basecoin/errors"
+)
+
+const (
+	MaxMembers = 10
+)
+
```

Comment:
good point, i was thinking that the devs of basecoin/quark find a reasonable number, but yeah, maybe it is a chain-dependent number.

i'll make it a var